TAF/utils/scripts/docker/get-compose-file-perfermance.sh

```diff
@@ -10,10 +10,10 @@ USE_SECURITY=${2:--}
 [ "$USE_SECURITY" != '-security-' ] && USE_NO_SECURITY="-no-secty"
 
 # # sync docker-compose file from developer-script repo
-./sync-nightly-build.sh master ${USE_NO_SECURITY} ${USE_ARM64}
+./sync-nightly-build.sh master "${USE_NO_SECURITY}" "${USE_ARM64}" "-perf"
 
-cp docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose.yaml
+cp docker-compose-taf-perf-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose-mqtt.yaml
 
-# Insert required services for retrieving exported time
-sed -e '1,/- edgex-network/d' docker-compose-end-to-end.yaml > docker-compose-end-mqtt.yaml
-sed -e '/app-service-rules:/r docker-compose-end-mqtt.yaml' -e //N docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml > docker-compose-mqtt.yaml
+sed -i 's/\EXPORT_HOST_PLACE_HOLDER/${DOCKER_HOST_IP}/g' docker-compose-mqtt.yaml
+sed -i 's/\MQTT_BROKER_ADDRESS_PLACE_HOLDER/${MQTT_BROKER_IP}/g' docker-compose-mqtt.yaml
```

Comment:
Please use `edgex-mqtt-broker` to instead of `${MQTT_BROKER_IP}`.

Reply:
Done